tables/it-it-comp6.utb

```diff
@@ -20,151 +34,514 @@
+base uppercase \x00c0 \x00e0 # a accent grave (Àà)
+base uppercase \x00c1 \x00e1 # a con accento acuto (Áá)
+base uppercase \x00c2 \x00e2 # a accent circonflexe (Ââ)
+base uppercase \x00c4 \x00e4 # a con dieresi (Ää)
+base uppercase \x00c6 \x00E6 # a e-dans-l’a (Ææ)
+base uppercase \x00c7 \x00e7 # c con cediglia (Çç)
+base uppercase \x00c8 \x00e8 # e con accento grave (Èè)
+base uppercase \x00c9 \x00e9 # e con accento acuto (Éé)
+base uppercase \x00ca \x00ea # e con accento circonflesso (Êê)
+base uppercase \x00cb \x00eb # e con dieresi (Ëë)
+base uppercase \x00cc \x00ec # i con accento grave (Ìì)
+base uppercase \x00cd \x00ed # i con accento acuto (Íí)
+base uppercase \x00ce \x00ee # i con accento circonflesso (Îî)
+base uppercase \x00cf \x00ef # i con dieresi (Ïï)
+base uppercase \x00d1 \x00f1 # n con tilde (Ññ)
+base uppercase \x00d2 \x00f2 # o con accento grave (Òò)
+base uppercase \x00d3 \x00f3 # o con accento acuto (Óó)
+base uppercase \x00d4 \x00f4 # o con accento circonflesso (Ôô)
+base uppercase \x00d6 \x00f6 # o con dieresi (Öö)
+base uppercase \x00d8 \x00f8 # o barrata (non ufficiale)
+base uppercase \x00d9 \x00f9 # u con accento grave (Ùù)
+base uppercase \x00da \x00fa # u con accento acuto (Úú)
+base uppercase \x00db \x00fb # u con accento circonflesso (Ûû)
+base uppercase \x00dc \x00fc # u con dieresi (Üü)
+base uppercase \x0152 \x0153 # oe legate (Œœ)
+base uppercase \x0391 \x03b1 Α,α alfa
+base uppercase \x0392 \x03b2 Β,β bêta
+base uppercase \x0393 \x03b3 Γ,γ gamma
+base uppercase \x0394 \x03b4 Δ,δ delta
+base uppercase \x0395 \x03B5 Ε,ε epsilon
+base uppercase \x0396 \x03B6 Ζ,ζ zêta
+base uppercase \x0397 \x03B7 Η,η êta
+base uppercase \x0398 \x03B8 Θ,θ thêta
+base uppercase \x0399 \x03B9 Ι,ι iota
+base uppercase \x039A \x03BA Κ,κ kappa
+base uppercase \x039B \x03BB Λ,λ lambda
+base uppercase \x039C \x03BC Μ,μ mu
+base uppercase \x039D \x03BD Ν,ν nu
+base uppercase \x039E \x03BE Ξ,ξ ksi/xi
+base uppercase \x039F \x03BF Ο,ο omicron
+base uppercase \x03A0 \x03C0 Π,π pi
+base uppercase \x03A1 \x03C1 Ρ,ρ rhô
+base uppercase \x03A3 \x03C3 Σ,σ sigma
+base uppercase \x03A4 \x03C4 Τ,τ tau
+base uppercase \x03A5 \x03C5 Υ,υ upsilon
+base uppercase \x03A6 \x03C6 Φ,φ phi
+base uppercase \x03A7 \x03C7 Χ,χ khi
+base uppercase \x03A8 \x03C8 Ψ,ψ psi
+base uppercase \x03A9 \x03C9 Ω,ω oméga
+
+# Uppercase letters in computer braille mode
+comp6 \x0391 46-45-1
+comp6 \x0392 46-45-12
+comp6 \x0393 46-45-1245
+comp6 \x0394 46-45-145
+comp6 \x0395 46-45-15
+comp6 \x0396 46-45-1356
+comp6 \x0397 46-45-125
+comp6 \x0398 46-45-245
+comp6 \x0399 46-45-24
+comp6 \x039A 46-45-13
+comp6 \x039B 46-45-123
+comp6 \x039C 46-45-134
+comp6 \x039D 46-45-1345
+comp6 \x039E 46-45-1346
+comp6 \x039F 46-45-135
+comp6 \x03A0 46-45-1234
+comp6 \x03A1 46-45-1235
+comp6 \x03A3 46-45-234
+comp6 \x03A4 46-45-2345
+comp6 \x03A5 46-45-136
+comp6 \x03A6 46-45-124
+comp6 \x03A7 46-45-12345
+comp6 \x03A8 46-45-13456
+comp6 \x03A9 46-45-2456
+
+# other signs
```

Comment:
From this point on, it starts to become clear that fr-bfu-comp-6.utb was used as a starting point. I would like to make sure that we're not leaving anything in this table that was not put or left there on purpose.